featuretests/cmd_juju_subnet_test.go

```diff
@@ -81,9 +81,8 @@ func (s *cmdSubnetSuite) TestSubnetAddCIDRAndInvalidSpaceName(c *gc.C) {
 }
 
 func (s *cmdSubnetSuite) TestSubnetAddAlreadyExistingCIDR(c *gc.C) {
-	c.Skip("Temporary for subnet cidr to id change.")
 	s.AddSpace(c, "foo", nil, true)
-	s.AddSubnet(c, network.SubnetInfo{CIDR: "0.10.0.0/24", SpaceName: "foo"})
+	s.AddSubnet(c, network.SubnetInfo{CIDR: "0.10.0.0/24", SpaceName: "foo", ProviderId: "dummy-private"})
 
 	expectedError := `cannot add subnet: adding subnet "0.10.0.0/24": subnet "0.10.0.0/24" already exists`
```

Comment:
We should consider including the provider ID in these messages.

Reply:
I have a card up to audit juju add-subnet, add this note to the card.